build-system/pr-check.js

```diff
@@ -380,21 +380,6 @@ function main(argv) {
     process.exit(1);
   }
 
-  // Make sure changes to package.json also update yarn.lock.
```

Comment:
Instead of removing this check, maybe repurpose it to make sure that `package.json` and `package-lock.json` are updated in lock step? (Heh!)

Reply:
Fair enough, PTAL